webapps-schema/src/main/java/io/camunda/webapps/schema/descriptors/usermanagement/index/TenantIndex.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.schema.descriptors.usermanagement.index;
+
+import io.camunda.webapps.schema.descriptors.usermanagement.UserManagementIndexDescriptor;
+
+public class TenantIndex extends UserManagementIndexDescriptor {
+  public static final String INDEX_NAME = "tenants";
+  public static final String INDEX_VERSION = "8.7.0";
+
+  public static final String TENANT_KEY = "tenantKey";
+  public static final String TENANT_ID = "tenantId";
+  public static final String NAME = "name";
+  public static final String ENTITY_KEY = "entityKey";
+  public static final String ENTITY_TYPE = "entityType";
```

Comment:
What do we use these static fields for? I've also noticed them in the other index descriptors, but they almost all seem to be unused.

Reply:
Very good question, I introduced them initially in other index classes but that was more for alignment (and at a time when the new Exporter code was new to me :D ) - I agree actually though that for our implementations here, these fields seem irrelevant (in the older index classes i.e. DecisionIndex the static variables are used elsewhere in the code.

Reply:
Do nothing for now and we can add in a clean up PR if necessary